### PR TITLE
FYLE-85zrk1mx7 Code coverage : Mask number pipe

### PR DESCRIPTION
**Pull request by CODEX108**

### CODE REVIEW

File: src/app/shared/pipes/mask-number.pipe.spec.ts [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
   it('create an instance', () => {
-    const pipe = new MaskNumber();
-    expect(pipe).toBeTruthy();
+    expect(maskNumber).toBeTruthy();
+  });
+
+  it('MaskNumber transform() : should mask the numbers and only display the last 4 digits', () => {
+    expect(maskNumber.transform(number1)).toBe('****6789');
+    expect(maskNumber.transform(number2)).toBe('****1234');
+    expect(maskNumber.transform('1')).toBe('****1');
```
Comment (jayfyle) [inline src/app/shared/pipes/mask-number.pipe.spec.ts:16]: I don't think check for single digits are needed, check @Dimple16 , maybe there not required.

**Review by jayfyle — APPROVED** [review]
Check for the single digit checks, if there not required remove them, rest looks good.

File: src/app/shared/pipes/mask-number.pipe.spec.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/shared/pipes/mask-number.pipe.spec.ts:16]: single digit check is not required, can remove it

**Review by Dimple16 — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]
minor comment, fix and merge

**Review by jayfyle — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**